plugins/source/aws/resources/services/backup/global_settings_fetch.go

```diff
@@ -15,14 +15,6 @@ func fetchBackupGlobalSettings(ctx context.Context, meta schema.ClientMeta, pare
 
 	output, err := svc.DescribeGlobalSettings(ctx, &input)
 	if err != nil {
-		if client.IgnoreAccessDeniedServiceDisabled(err) || client.IsAWSError(err, "ERROR_9601") /* "Your account is not a member of an organization" */ {
```

Comment:
How about we keep this change (and other removals of log messages/errors) to a separate PR.
Basically only do the `meta.Logger()` -> `meta.(*client.Client).Logger()` changes to keep the size of the PR small?

We can do a follow up to look into the error handling in various AWS resources and see how we want to handle them? I agree that it looks like we can remove most of them, but I can't be sure for a few

Reply:
I think it's a nit, it's already split. Let's keep like this and address other issue with AWS later on with unneeded/un-normalized log messages and so on.